src/sass_context.cpp

```diff
@@ -112,7 +112,7 @@ extern "C" {
       c_ctx->error_file = sass_strdup(e.pstate.path);
       c_ctx->error_line = e.pstate.line+1;
       c_ctx->error_column = e.pstate.column+1;
-      c_ctx->error_src = e.pstate.src;
+      c_ctx->error_src = sass_strdup(e.pstate.src);
```

Comment:
Added a free call in 9874428. I think no other down-stream is using it. When we get the syntax error and call `sass_context_get_error_src` after the compilation, it returns garbage, because libsass deletes cpp_ctx right after compilation. Caught this when debugging.